rust/benches/reader.rs

```diff
@@ -0,0 +1,90 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use mcap::{Channel, Message, MessageStream, Schema};
+use std::borrow::Cow;
+use std::io::Cursor;
+use std::sync::Arc;
+use std::time::Duration;
+
+fn create_test_mcap(n: usize, compression: Option<mcap::Compression>) -> Vec<u8> {
+    let mut buffer = Vec::new();
+    {
+        let mut writer = mcap::WriteOptions::new()
+            .compression(compression)
+            .profile("fooey")
+            .create(Cursor::new(&mut buffer))
+            .unwrap();
+        const MESSAGE_DATA: &[u8] = &[0; 42];
```

Comment:
any reason for 42 here?

Reply:
It was derived from:
https://github.com/foxglove/mcap/blob/c95275f41fc931f3f444c3d3b9edccb4d677c17d/typescript/benchmarks/index.ts#L40

To give the bench some message-data so cross language benches can somewhat be compared apples-to-apples, but it should instead be `[42; 10]` (10 times `42` instead of 42 zero bytes).

I'll fix that, good catch